plugin-server/src/cdp/templates/test/test-helpers.ts

```diff
@@ -245,3 +247,37 @@ export class TemplateTester {
         return this.executor.execute(modifiedInvocation)
     }
 }
+
+export const createPayload = (
```

Comment:
`createPayload` is super generic. Can we make it much more explicit please (also makes auto-importing much more likely to work)

Reply:
Makes sense - Renamed the function to `createAdDestinationPayload`